openpype/hosts/maya/plugins/publish/extract_look.py

```diff
@@ -42,6 +43,58 @@ def find_paths_by_hash(texture_hash):
     return io.distinct(key, {"type": "version"})
 
 
+def rstex(source, *args):
+    """Make `.rstexbin` using `redshiftTextureProcessor`
+    with some default settings.
+
+    This function requires the `REDSHIFT_COREDATAPATH`
+    to be in `PATH`.
+
+    Args:
+        source (str): Path to source file.
+        *args: Additional arguments for `redshiftTextureProcessor`.
+
+    Returns:
+        str: Output of `redshiftTextureProcessor` command.
+
+    """
+    if "REDSHIFT_COREDATAPATH" not in os.environ:
```

Comment:
Note: I think we should validate this environment during validation, only when is needed if it's possible to know.

Reply:
Ah, I see! Thank you, I'll take a look about moving that

Reply:
That's something I don't know. Maybe it's not needed or impossible to validate before this plugin, can't tell.

Reply:
This is more complex as this is set only if studio is using and has configured Redshift. So I think the logic should be - add option `make redshift texture` on the same place as we have with make tx (look instance) and then probably have validator that will run on look family and will validate this env var.